### PR TITLE
updated index.html and main.css

### PR DESCRIPTION
**Pull request by Kanhalelor**

### CODE REVIEW

**Comment by Kanhalelor** [comment]
*More changes will be coming soon!*

**Comment by windhoeknamibia** [comment]
**LGTM**

**Review by windhoeknamibia — COMMENTED** [review]
PR Approved

**Review by windhoeknamibia — COMMENTED** [review]
PR Approved

File: css/main.css [hunk 1]
```diff
@@ -24,11 +24,9 @@ h1, h2, h3, h4 {
 }
 
 #title {
-    color: rgb(228, 218, 216);
-    padding-top: 20px;
+    color: white;
```
Comment (windhoeknamibia) [inline css/main.css:27]: Good

**Comment by Kanhalelor** [comment]
Please merge changes

File: css/main.css (hunk 1 above)
Comment (windhoeknamibia) [inline css/main.css:27]: Great

File: index.html [hunk 2]
```diff
@@ -22,118 +22,133 @@ <h4 class="place-name">Independence Museum</h4>
                     <img src="https://momaa.org/wp-content/uploads/2019/10/Independence-Memorial-Museum-Namibia.png" alt="windhoek museum" class="place-img">
                     <div class="place-link">
                         <ul>
-                        <li><a href="#">VISIT</a></li>
-                        <li><a href="#">LEARN MORE</a></li>
+                        <li><a href="https://www.google.com/maps/place/Independence+Museum/@-22.5688266,17.0879955,15z/data=!4m5!3m4!1s0x0:0x9964518a738580b!8m2!3d-22.5688266!4d17.0879955">VISIT</a></li>
+                        <li><a href="https://en.wikipedia.org/wiki/Independence_Memorial_Museum_(Namibia)">LEARN MORE</a></li>
```
Comment (windhoeknamibia) [inline index.html:26]: LGTM

File: index.html [hunk 3]
```diff
@@ -22,118 +22,133 @@ <h4 class="place-name">Independence Museum</h4>
...
                         <ul>
-                        <li><a href="#">VISIT</a></li>
-                        <li><a href="#">LEARN MORE</a></li>
+                        <li><a href="https://www.google.com/maps/place/University+of+Namibia/@-22.6121691,17.0583726,15z/data=!4m5!3m4!1s0x0:0x8b4b657accaf67dd!8m2!3d-22.6121691!4d17.0583726">VISIT</a></li>
+                        <li><a href="https://en.wikipedia.org/wiki/University_of_Namibia">LEARN MORE</a></li>
                     </ul>
                     </div>
                 </div>
                 <div class="whk-place">
-                    <h4 class="place-name">Independence Museum</h4>
```
Comment (windhoeknamibia) [inline index.html:131]: Will fix this typo

**Review by windhoeknamibia — APPROVED** [review]

**Pull request merged**